pkg/storage/ingest/partition_offset_reader.go

```diff
@@ -120,10 +120,12 @@ func (p *partitionOffsetReader) getAndNotifyLastProducedOffset(ctx context.Conte
 	promise.notify(offset, err)
 }
 
-// getLastProducedOffset fetches and returns the last produced offset for a partition, or -1 if the
+// RequestLastProducedOffset fetches and returns the last produced offset for a partition, or -1 if the
 // partition is empty. This function issues a single request, but the Kafka client used under the
 // hood may retry a failed request until the retry timeout is hit.
-func (p *partitionOffsetReader) getLastProducedOffset(ctx context.Context) (_ int64, returnErr error) {
+//
+// This function may be directly called from outside the partitionOffsetReader.
```

Comment:
i don't understand this comment. Doesn't the exported method imply  this?

Reply:
Removed.